heartbeat/nfsserver

```diff
@@ -5,6 +5,13 @@
 # by hxinwei@gmail.com
 # License: GNU General Public License v2 (GPLv2) and later
 
+
+# I don't know for certain whether all services actuall _need_ this,
+# I know that at least nfs-server needs it.
+NFSD_RELATED_SYSTEMD_SERVICE_FOR_UNSHARE_UTS_NAMESPACE="nfs-idmapd.service nfs-mountd.service nfs-server.service nfsdcld.service rpc-gssd.service rpc-statd.service rpcbind.service"
```

Comment:
I can't think of a reason why you would need to change the uts namespace for any of those services besides nfs-server.service.  The rest all start userspace daemons, and none of them call uname() except for rpc.mountd (which does it to get the kernel version).

Reply:
if you look at [the rgmanager agent](https://github.com/ClusterLabs/resource-agents/blob/8dc8b40b69d522400e72c883e60e90ddb6392680/rgmanager/src/resources/nfsserver.sh.in#L224), nfs, rpcgssd and rpcidmapd have the unshare. Sure, it could be unnecessary there as well, but to me that sounds plausible at least.

Reply:
Let's try to do it for nfs-server.service, and just have the above list as a comment, so we can add any of those if we see any issues.

It's best to try and minimize the amount of changes we do to the system.